wrench/src/yaml_helper.rs

```diff
@@ -47,6 +48,28 @@ fn string_to_color(color: &str) -> Option<ColorF> {
     }
 }
 
+fn string_to_mix_blend_mode(mix_blend_mode: &str) -> Option<MixBlendMode> {
```

Comment:
I suggest we make a constant list of pairs here, and then just walk it up back and forth for conversion between strings and blend modes. That would avoid repetition (DRY FTW).